src/chargen.cpp

```diff
@@ -1126,8 +1126,12 @@ static void start_game(descriptor_data *d, const char *origin)
     GET_LOADROOM(d->character) = RM_CHARGEN_START_ROOM;
   }
 
-  // Otaku get some starting gear, so assign that here
+  // Otaku get some starting gear and skills, so assign that here
   if (d->ccr.is_otaku) {
+    // Assign starting computers to be helpful
+    GET_SKILL_POINTS(d->character) -= 6;

```

Comment:
What if someone has less than 6 skill points at creation? Do you enforce the 6-point minimum?

Reply:
Have to. :/ That's the rules. The south exit on chargen won't let you proceed with negative skillpoints too.

Reply:
I believe they can sell english skill points though to afford it.

Reply:
Can we force folks to have at least 6 skill points when moving from the creation menus to the game? I don't want folks cheesing it by unlearning English to meet their minimum.